code/modules/mob/inventory.dm

```diff
@@ -103,15 +103,17 @@
 	if(!I)
 		return 1
 	if((I.flags & NODROP) && !force)
-		return 0
+		return
+	if(I.flags & ABSTRACT) //An abstract item should never be dropped, even if it's being forced
+		return
 	return 1
 
 /mob/proc/unEquip(obj/item/I, force) //Force overrides NODROP for things like wizarditis and admin undress.
 	if(!I) //If there's nothing to drop, the drop is automatically succesfull. If(unEquip) should generally be used to check for NODROP.
 		return 1
 
 	if(!canUnEquip(I, force))
-		return 0
+		return

```

Comment:
Explicitly return a 0, please, this proc is used as a conditional in some cases.